zerver/views/invite.py

```diff
@@ -51,6 +51,34 @@ def invite_users_backend(request, user_profile,
     do_invite_users(user_profile, invitee_emails, streams, body)
     return json_success()
 
+@require_realm_admin    
+def get_user_invites(request, user_profile):
+    all_users = do_get_user_invites()
+    #ret = {'invites': all_users,
+    #    "result": "success",
+    #    "msg": ""}
+    return json_response(data={'invites': all_users})
+    
+@require_realm_admin
+@has_request_variables
+def revoke_user_invite(request, user_profile, prereg_id=REQ(converter=to_non_negative_int)):
+    if do_revoke_user_invite(prereg_id):
+        return json_success()
+    else:
+        return json_error(_("Cannot revoke the invitation, the invite was not found."))
```

Comment:
For this error condition, you may just want to have `do_revoke_user_invite` raise a `JsonableError`, rather than returning a success/failure status code.